_meta/short_description.md

```diff
@@ -0,0 +1,2 @@
+This workshop teaches participants to make their research software more reproducible with Docker 
+containers.
```

Comment:
Make clear that this can be max 140 characters

Reply:
Fixed. I just forgot to save my changes after editing 😅